File: Semestrul 4/Medii de proiectare si programare/Laboratoare/FirmaTransportJava/Networking/src/main/java/transport/networking/dto/SeatDTO.java
package transport.networking.dto;

import java.io.Serializable;

public class SeatDTO implements Serializable {
    private int number;
    private String clientName;

    public SeatDTO(int number, String clientName) {
        this.number = number;
        this.clientName = clientName;
    }

    public int getNumber() {
        return number;
    }

    public String getClientName() {
        return clientName;
    }
}

File: Semestrul 4/Medii de proiectare si programare/Laboratoare/FirmaTransportC#/Network/dto/SeatDTO.cs
using System.Text.Json.Serialization;

namespace FirmaTransportC_.Networking.dto
{
    [Serializable]
    public class SeatDTO
    {
        public long Id { get; set; }
        public int Number { get; set; }
        public long TripId { get; set; }
        public long? ReservationId { get; set; }
        public string? ClientName { get; set; }

        public SeatDTO(long id, int number, long tripId, long? reservationId, string? clientName)
        {
            Id = id;
            Number = number;
            TripId = tripId;
            ReservationId = reservationId;
            ClientName = clientName;
        }

        public SeatDTO()
        {
        }
    }
}
